fam/llm/serving.py

```diff
@@ -161,7 +166,7 @@ def _convert_audiodata_to_wav_path(audiodata, wav_tmp):
         seed=1337,
         device=device,
         dtype=GlobalState.config.dtype,
-        compile=False,
+        compile=GlobalState.config.compile,
```

Comment:
this will not work right now and cause recompilations at each time-steps

Reply:
it does not seem to do that for me

Reply:
did you try with vanilla kv-cache or flash decoding?

Reply:
also, we probably need to change the mode for `torch.compile` during inference to get the most out of it, and to use cuda graphs